gcc/d/patches/patch-gcc-8.x

```diff
@@ -2,6 +2,23 @@ This implements D language support in the GCC back end, and adds
 relevant documentation about the GDC front end.
 ---
 
+--- a/gcc/config/powerpcspe/powerpcspe.c
++++ b/gcc/config/powerpcspe/powerpcspe.c
```

Comment:
Also applying this patch from `rs6000.c` to `powerpcspe.c`, as the later is (mostly) a direct copy from the former.